### PR TITLE
second sprint version - get data only from geoserver for now

### PR DESCRIPTION
**Pull request by dafnaweiss**

### CODE REVIEW

File: src/components/LayersDataTable.tsx [hunk 1]
```diff
@@ -36,7 +38,9 @@ class LayersDataTable extends React.Component {
         }
     };
 
-    state: any  = {};
+    state: any  = {
+        displayProgressBar: false
```
Comment (yairtawil) [inline src/components/LayersDataTable.tsx]: displayDailog

File: src/components/LayersDataTable.tsx [hunk 2]
```diff
@@ -217,16 +238,16 @@ class LayersDataTable extends React.Component {
...
                                 </div>
                             </div>
                             <div className="ui-grid-row">
                                 <div className="ui-grid-col-4" style={{padding:'4px 10px'}}><label htmlFor="id">Bounding Box</label></div>
                                 <div className="ui-grid-col-8" style={{padding:'4px 10px'}}>
-                                    <InputText id="nativeBoundingBoxminX" value={this.selectedLayerType.nativeBoundingBox.minx}/>
-                                    <InputText id="nativeBoundingBoxminX" value={this.selectedLayerType.nativeBoundingBox.maxx}/>
-                                    <InputText id="nativeBoundingBoxmaxY" value={this.selectedLayerType.nativeBoundingBox.miny}/>
-                                    <InputText id="nativeBoundingBoxminY" value={this.selectedLayerType.nativeBoundingBox.maxy}/>
+                                    <InputText id="nativeBoundingBoxminX" value={this.selectedLayerType.nativeBoundingBox.minx} readOnly={true}/>
```
Comment (yairtawil) [inline src/components/LayersDataTable.tsx]: map ?
remove

File: src/components/ProgressBarComponent.tsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+/* Prime React components */
+import 'primereact/resources/primereact.min.css';
+import 'primereact/resources/themes/omega/theme.css';
+import 'font-awesome/css/font-awesome.css';
+import { ProgressBar } from 'primereact/components/progressbar/ProgressBar';
+// import {Growl} from 'primereact/components/growl/Growl';
+
+class ProgressBarComponent extends React.Component {
+    props: any;
```
Comment (yairtawil) [inline src/components/ProgressBarComponent.tsx]: interface

File: src/components/ProgressBarComponent.tsx [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    state = {value1: 0, value2: 50};
```
Comment (yairtawil) [inline src/components/ProgressBarComponent.tsx]: lint

File: src/components/ProgressBarComponent.tsx [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+            }
+            this.setState({value1: val});
+        }, 2000);
+    }
+
+    render() {
+        return (
+            <div>
+                <div className="content-section implementation">
+                    <ProgressBar value={this.state.value1}/>
```
Comment (yairtawil) [inline src/components/ProgressBarComponent.tsx]: if 100 hide

File: src/components/UploadFile.tsx [hunk 6]
```diff
@@ -91,3 +82,8 @@ const mapDispatchToProps = (dispatch: any) => ({
 export default connect(mapStateToProps, mapDispatchToProps)(UploadFile);
 
 // withCredentials={true}
```
Comment (yairtawil) [inline src/components/UploadFile.tsx]: remvoe

File: src/services/UploadFileService.ts [hunk 7]
```diff
@@ -0,0 +1,19 @@
+import axios from 'axios';
+import config from "../config/config";
+
+export class UploadFileService {
+
+    static urlBase: string = `http://${config.ipAddress}${config.serverPort}/api/upload/`;
```
Comment (yairtawil) [inline src/services/UploadFileService.ts]: config?
baseUrl

File: src/services/UploadFileService.ts [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+    // ==============
+    // get all layers of the world (including the ILayer's fields)
+    static upload(worldName: string, file: File): Promise<any> {
+        console.log("start the upload file service...");
+        return axios
+            .post(`${this.urlBase}${worldName}`, file)
+            .then(res => console.log("the upload succeed!"))
+            .catch(error => console.log(error));
+    }
+}
```
Comment (yairtawil) [inline src/services/UploadFileService.ts]: enter

**Review by yairtawil — APPROVED** [review]

File: src/components/LayersDataTable.tsx (hunk 2 above)
Comment (dafnaweiss) [reply src/components/LayersDataTable.tsx]: I will remove it in the next version. all the table is going to be changed

File: src/components/ProgressBarComponent.tsx (hunk 4 above)
Comment (dafnaweiss) [reply src/components/ProgressBarComponent.tsx]: I removed this component. using the prime react component

File: src/components/ProgressBarComponent.tsx (hunk 5 above)
Comment (dafnaweiss) [reply src/components/ProgressBarComponent.tsx]: I removed this component. using the prime react component

**Pull request merged**